conntrack_linux.go

```diff
@@ -326,11 +326,11 @@ func parseRawData(data []byte) *ConntrackFlow {
 type ConntrackFilterType uint8
 
 const (
-	ConntrackOrigSrcIP = iota // -orig-src ip   Source address from original direction
```

Comment:
@fcrisciani The const you are changing are actually exported and were there before we cut `1.0.0`.
You may add the new ones and mark the existing ones as deprecated in their comment.